contracts/stake/src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub const MATURITY: u32 = 2 * EPOCH;
 pub const VALIDITY: u32 = 56 * EPOCH;
 
 /// The minimum amount of Dusk one can stake.
-pub const MINIMUM_STAKE: u64 = 5_000;
+pub const MINIMUM_STAKE: u64 = 5_000_000;
```

Comment:
5M dusk or 5M nanodusk?

Reply:
It should be 5k M *micro*dusk. Thanks for making me look twice. It needs three more zeros.

Reply:
Perfect, so it needs a change in the comment too